src/stage1/migrate_info/balena_cfg_json.rs

```diff
@@ -230,3 +198,33 @@ impl BalenaCfgJson {
         &self.file
     }
 }
+
+fn check_api(api_endpoint: &str) -> Result<()> {
+    let res = Client::builder()
+        .build()
+        .upstream_with_context("Failed to create https client")?
+        .get(api_endpoint)
```

Comment:
Isn't this URL coming directly from `config.json`? So, unless I missed something you'd need to append `/ping` here, right?

Reply:
nice catch! i had /ping in the test URL 😓